core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -35,11 +36,14 @@ import (
 )
 
 const (
+	defaultPluginRetryExpiration = 30 * time.Minute
 	// defaultAllowListExpiration decides how long an upkeep's allow list info will be valid for.
 	defaultAllowListExpiration = 20 * time.Minute
-	// allowListCleanupInterval decides when the expired items in allowList cache will be deleted.
-	allowListCleanupInterval   = 5 * time.Minute
+	// cleanupInterval decides when the expired items in cache will be deleted.
+	cleanupInterval            = 5 * time.Minute
 	logTriggerRefreshBatchSize = 32
+	totalFastPluginRetries     = 5
+	totalPluginRetries         = 10
```

Comment:
nit. not sure this is actually "total Retries" ? maybe totalMediumPluginRetries?